pkgs/development/libraries/rnnoise-plugin/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, SDL2, fetchFromGitHub, cmake }:
+
+
+stdenv.mkDerivation rec {
+  name = "rnnoise-plugin-${version}";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "werman";
+    repo = "noise-suppression-for-voice";
+    rev =  "v${version}";
+    sha256 = "18bq5b50xw3d4r1ildinafpg3isb9y216430h4mm9wr3ir7h76a7";
+  };
+
+
+  buildInputs = [ cmake ];
+
+ 
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
+
+  meta = with stdenv.lib; {
+    description = "A real-time noise suppression plugin for voice based on Xiph's RNNoise.";
```

```suggestion
    description = "A real-time noise suppression plugin for voice based on Xiph's RNNoise";
```

Small nitpick: nixpkgs contribution guidelines require that there is no period at the end of the description